app/models/folio/patron.rb

```diff
@@ -159,6 +172,12 @@ def all_proxy_group_info
       @all_proxy_group_info ||= self.class.folio_client.all_proxy_group_info(id)
     end
 
+    # Get all the sponsors for which this patron is a proxy
+    def all_proxy_info
+      @all_proxy_info ||= user_info.dig('stubs', 'all_proxy_info') # used for stubbing
```

Comment:
what does "used for stubbing" mean here? it doesn't look like this behavior gets used in the test added for this PR, but I'm not 100% sure

Reply:
ah, I see – this PR didn't change anything about this. thanks